sdk/textanalytics/Azure.AI.TextAnalytics/src/OperationContinuationToken.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using Azure.Core;
+
+using ServiceVersion = Azure.AI.TextAnalytics.TextAnalyticsClientOptions.ServiceVersion;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal class OperationContinuationToken
+    {
+        private const ServiceVersion ContinuationTokenVersion = ServiceVersion.V3_1;
+
+        // Parameterless constructor required for JSON deserialization.
+        public OperationContinuationToken()
+        {
+            Version = TextAnalyticsClientOptions.GetVersionString(ContinuationTokenVersion);
+        }
+
+        public OperationContinuationToken(string jobId, IDictionary<string, int> inputDocumentOrder, bool? showStats)
+            : this()
+        {
+            JobId = jobId;
+            InputDocumentOrder = new Dictionary<string, int>(inputDocumentOrder);
+            ShowStats = showStats;
+        }
+
+        public string Version { get; set; }
```

Comment:
What's the casing on these look like when serialized?  `Version` or `version`?  We typically camelCase JSON property names so `version` would be preferable.  We could do that either by adding [`[JsonPropertyName("version")]`](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-customize-properties?pivots=dotnet-6-0#customize-individual-property-names) on all these members or [passing an instance of `JsonSerializerOptions` with `PropertyNamingPolicy` set to `CamelCase`](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-customize-properties?pivots=dotnet-6-0#use-camel-case-for-all-json-property-names).

Reply:
Updated code to use `PropertyNamingPolicy.CamelCase` when (de)serializing.